src/tools/Brush.js

```diff
@@ -182,9 +182,15 @@ const _Tool = types
         )
           return;
         const c = self.control;
+        const o = self.obj;
 
-        // Reset the timer if a user started drawing again
         brush = self.getSelectedShape;
+
+        // prevent drawing when current image is
+        // different from image where the brush was started
+        if (o && brush && o.multiImage && o.currentImage !== brush.item_index) return;
```

Comment:
Here is the same problem with `brush.item_index===null`. I'm not sure why it's happening yet.